pkg/controller/v1beta1/inferenceservice/components/explainer.go

```diff
@@ -71,8 +71,12 @@ func NewExplainer(client client.Client, clientset kubernetes.Interface, scheme *
 func (e *Explainer) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, error) {
 	e.Log.Info("Reconciling Explainer", "ExplainerSpec", isvc.Spec.Explainer)
 	explainer := isvc.Spec.Explainer.GetImplementation()
+	disallowedAnnotations := constants.ServiceAnnotationDisallowedList
+	if e.deploymentMode == constants.RawDeployment {
+		disallowedAnnotations = constants.RawServiceAnnotationDisallowedList
+	}
 	annotations := utils.Filter(isvc.Annotations, func(key string) bool {
-		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
+		return !utils.Includes(disallowedAnnotations, key)
```

Comment:
I believe these changes will have conflicts, it should be something like:
```
return !utils.Includes(e.inferenceServiceConfig.ServiceAnnotationDisallowedList, key)
```

as the sync was merged.
Can you please rebase?

Reply:
fixed